changelog/1582.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
```

Comment:
I'd suggest removing this file :-) No error strings were impacted and most other things shouldn't be materially impacted IMHO.